src/Utility/TypeExtensions.cs

```diff
@@ -129,8 +146,13 @@ static RpcHttp ToRpcHttp (this HttpResponseContext httpResponseContext)
             return rpcHttp;
         }
 
-        public static TypedData ToTypedData(this object value)
+        internal static TypedData ToTypedData(this object value, PowerShellManager psHelper)
```

Comment:
I'm not sure how I feel about having to pass in the psHelper all the time just so that we can reuse the powershell session to run `ConvertTo-Json`... 

Can we just create a constant PowerShell instance in this file that's only used to call `ConvertTo-Json`?